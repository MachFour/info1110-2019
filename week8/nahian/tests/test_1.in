2
spam
-7
5
