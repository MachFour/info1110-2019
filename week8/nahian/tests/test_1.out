Enter a: Enter b: Enter b: Enter c: 
The equation 2x^2 + -7x + 5 = 0 has solutions:
x=2.50, x=1.00
